apps/OpenSpace/ext/launcher/src/sgctedit/sgctedit.cpp

```diff
@@ -277,6 +277,11 @@ sgct::config::Cluster SgctEdit::generateConfiguration() const {
         window.tags.push_back("GUI");
         window.draw2D = true;
         window.draw3D = false;
+
+        //Disable 2D rendering on all non-GUI windows
+        for (unsigned int w = 1; w < node.windows.size(); ++w) {
```

Comment:
More modern version of this:
```cpp
for (sgct::Window& w : node.windows) {
    w.draw2D = false;
}
```

Reply:
I also like using that version, but I did the for loop because I need to skip past the first element of `node.windows` since it has the GUI.

Reply:
Do'h!  Yea, I missed that.   In that case it should be a `size_t` instead of an `unsigned int` anyway since that's the return type of the `size` function